lib/modules/manager/bitbucket-pipelines/readme.md

```diff
@@ -1,3 +1,50 @@
 Extracts dependencies from Bitbucket Pipelines config files.
 
 If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+
+### Using private build images
+
+Bitbucket Pipelines supports using an authenticated Docker registry, including self-hosted private registries, Docker Hub, Amazon ECR and Google GCR.
+
+More details can be found on the Atlassian [Bitbucket Pipeline documentation](https://support.atlassian.com/bitbucket-cloud/docs/use-docker-images-as-build-environments) site
```

Comment:
```suggestion
More details can be found on the Atlassian [Bitbucket Pipeline documentation](https://support.atlassian.com/bitbucket-cloud/docs/use-docker-images-as-build-environments) site.
```

Reply:
Thanks @HonkingGoose 

I'll be updating this PR one @viceice can land https://github.com/renovatebot/renovate/pull/21102